girder/stemserver_plugin/models/stemimage.py

```diff
@@ -93,24 +94,30 @@ def delete(self, id, user):
 
         return self.remove(stem_image)
 
-    def _get_file(self, stem_image_id, user):
-        """Get a file object of the stem image"""
+    @contextmanager
+    def _get_h5py_file(self, stem_image_id, user):
```

Comment:
Would it make more sense if it was `_open_h5py_file`?

Reply:
Done.